### ISSUE TITLE
[READ ME] Apache 2.4 not working for Debian Wheezy; Dropping support for Debian incoming to PuPHPet

### ISSUE THREAD

**Issue by django23** [puphpet/puphpet#2436]
Dear Puphpet,

It seems that d7031.de isn't available anymore. The site seems down and for that reason it's not possible to install any instance of apache.

`]  
class puphpet::apache::repo::debian {

 apt::source { 'd7031.de':
   location          => 'http://www.d7031.de/debian/',
   release           => 'wheezy-experimental',
   repos             => 'main',
   required_packages => 'debian-keyring debian-archive-keyring',
   key               => {
     'id'      => '9EB5E8A3DF17D0B3',
     'server'  => 'hkp://keyserver.ubuntu.com:80',
   },
   include           => { 'src' => true }
 }

}`

**Comment by jtreminio** [comment]
Back when I added Apache HTTPD 2.4 support I searched and searched for a Wheezy 2.4 repo; this was the only one I found. I am _extremely_ hesitant on managing my own repos for packages.

At this point, unless someone can point to me a _stable_ repo with Apache 2.4 for Wheezy, I am a day away from dropping Debian support completely from PuPHPet. Its single-minded devotion to stability makes adding features and software support to it a pain in the butt, one that has wasted hundreds of my hours and giving me more stress than I wanted.

I'm sure this isn't the answer you wanted, but it's the only one I can give and remain sane and not burned out supporting this project.
